client/src/components/Form/FormRepeat.vue

```diff
@@ -128,16 +153,18 @@ const { keyObject } = useKeyedObjects();
                         </b-button>
                     </b-button-group>
 
-                    <b-button
-                        v-b-tooltip.hover.bottom
-                        title="delete"
-                        role="button"
-                        variant="link"
-                        size="sm"
-                        class="ml-0"
-                        @click="() => onDelete(cacheId)">
-                        <FontAwesomeIcon icon="trash-alt" />
-                    </b-button>
+                    <span v-b-tooltip.hover.bottom :title="deleteTooltip">
+                        <b-button
+                            :disabled="!minRepeats"
+                            title="delete"
+                            role="button"
+                            variant="link"
+                            size="sm"
+                            class="ml-0"
+                            @click="() => onDelete(cacheId)">
+                            <FontAwesomeIcon icon="trash-alt" />
+                        </b-button>
+                    </span>
```

Comment:
This tooltip won't show up for keyboard users, but the solution for that is convoluted, so I'm fine with leaving it as is for now, and keeping a note of adding accessible tooltips to the future component library directly.

As a reference:

The way to make accessible tool-tips on any form of input, is to never actually disable them. Instead use (aria-disabled)[https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled], select it with an attribute selector, and style the element to look disabled. also wrap click/input events to check if the input is "disabled", and only forward them if it is not. This still allows said elements to be navigated to with a keyboard, so the reason for the element being disabled can be communicated to screen reader and keyboard users.

Reply:
@ElectronicBlueberry Thank you - that's a really good creative solution to adding accessibility and perhaps worthy of its own Github issue for future improvements.